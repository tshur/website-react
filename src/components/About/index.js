import React from 'react';
import './index.scss';

const About = () =>
  <div className='container about'>
    <h1> About </h1>
  </div>

export default About;