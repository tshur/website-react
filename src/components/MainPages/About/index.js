import React from 'react';
import './index.scss';

const About = () =>
  <div className='page-container about'>
    <div className='container'>
      <div className='about-card card-shadow'>
        <h1 className='title'> About Me </h1>
        <p className='text'> Hi there! I am a passionate and highly-motivated
        software engineer with experience in software engineering, machine
        learning, and full-stack web development. I am currently a student at
        Santa Clara University studying Computer Science &amp; Engineering and
        Mathematics. I am grateful to have gotten the opportunity of working with
        incredible teams at Nike and Headspin doing research and engineering work.
        Lately, I have been excited to get more into React and develop my expertise
        with artificial intelligence in Python. Check out some of my recent projects
        to see what I've been up to! </p>
      </div>
    </div>
  </div>

export default About;