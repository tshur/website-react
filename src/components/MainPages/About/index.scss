.about {
    .container {
        .about-card {
            background: #ffffff;
            margin-top: 100px;
            margin-left: 10px;
            margin-right: 10px;
            padding: 40px;
            .title {
                text-transform: uppercase;
            }
            .text {
                margin: 0;
                font-size: 22px;
                line-height: 1.5;
            }
        }
    }
}