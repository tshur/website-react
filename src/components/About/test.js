import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import About from '.';

describe('About', () => {

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<About />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  test('has a valid snapshot', () => {
    const component = renderer.create(
      <About />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
