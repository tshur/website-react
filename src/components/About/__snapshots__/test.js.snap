// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About has a valid snapshot 1`] = `
<div
  className="container undefined"
>
  <h1>
     About 
  </h1>
</div>
`;
